integration/services/ha_backend/setup.sh

```diff
@@ -10,7 +10,7 @@ vm.dirty_ratio=20
 vm.dirty_background_ratio=30
 vm.dirty_expire_centisecs=30000
 net.ipv4.ip_local_port_range=1024 65024 net.ipv4.tcp_max_syn_backlog=60000
-net.ipv4.tcp_tw_reuse=1
+# net.ipv4.tcp_tw_reuse=1
```

Comment:
why is this commented out? Should we remove it?

Reply:
No, I will uncomment it for CI. I commented it out because docker on vagrant doesn't like it.